package-lock.json

```diff
@@ -3840,35 +16948,36 @@
             }
         },
         "extract-zip": {
-            "version": "2.0.1",
-            "resolved": "https://registry.npmjs.org/extract-zip/-/extract-zip-2.0.1.tgz",
-            "integrity": "sha512-GDhU9ntwuKyGXdZBUgTIe+vXnWj0fppUEtMDL0+idd5Sta8TGpHssn/eusA9mrPr9qNDym6SxAYZjNvCn/9RBg==",
-            "dev": true,
+            "version": "1.7.0",
```

Comment:
This is the problem with this file. We need the 2.0.1 version, but I don't think we should manually updating this file

Reply:
I realized I used the one from main but just updated it with our internship master one.

Reply:
Sounds good, I restored the `package-lock.json` file and now the only change in this PR is re-adding the `tasks.json` file.